android-app/app/src/main/res/values/strings.xml

```diff
@@ -97,5 +95,17 @@
     <string name="dashboard_filter_dialog_title">Nach Kategorien filtern</string>
     <string name="dashboard_filter_indication_categories">%1$o Kategorie(n) werden nicht angezeigt</string>
     <string name="apply">Übernehmen</string>
-    <string name="register_user_error">Ungültige Angaben gemacht!</string>
+    <string name="invalid_input_error">Ungültige Angaben gemacht!</string>
+    <string name="already_registered_question">Bereits registriert?</string>
+    <string name="click_here_for_login">Klicke hier zum einloggen!</string>
+    <string name="no_account_yet_question">Noch kein Konto?</string>
+    <string name="click_here_to_register">Jetzt Registrieren!</string>
+    <string name="login_button">Login</string>
+    <string name="login_text">Melde dich mit deinem Konto an</string>
+    <string name="login_title">Anmelden</string>
+    <string name="login_currently_unavailable">Login zur Zeit nicht möglich</string>
+    <string name="error_message_invalid_credentials">Ungültige Angaben</string>
+    <string name="error_message_server_error">Server Fehler</string>
+    <string name="error_message_unknown_error_occurred">Ein unbekannter Fehler ist aufgetreten</string>
+    <string name="registration_currently_unavailable">Registration zur Zeit nicht möglich</string>
```

Comment:
```suggestion
    <string name="registration_currently_unavailable">Registrierung zur Zeit nicht möglich</string>
```

personal taste, accept or don't accept as you like

Reply:
I agree it sounds better this way :)